lib/files.ex

```diff
@@ -31,11 +31,25 @@ defmodule Uploadex.Files do
   @spec store_files(record, Uploader.t) :: {:ok, record} | {:error, any()}
   def store_files(record, uploader) do
     files = wrap_files(record, uploader)
+    validate_and_store_files(record, files, uploader)
+  end
+
+  @spec store_files(record, Uploader.t, record) :: {:ok, record} | {:error, any()}
+  def store_files(record, uploader, previous_record) do
+    current_files = wrap_files(record, uploader)
+    previous_files = wrap_files(previous_record, uploader)
+
+    changed_files = current_files -- previous_files
```

Comment:
There is a very similar function in this module (`get_changed_files`) that does this but with the parameters inverted. What do you think about renaming it to something like `get_discarded_files` and also adding a new function `get_new_files`?

The goal is to make it clear what the functions are doing and why are they slightly different:

```elixir
defp get_discarded_files(new_files, old_files), do: old_files -- new_files

defp get_new_files(new_files, old_files), do: new_files -- old_files
```